Wrong android sdk checksum doesn't stop release process

* KOReader version: todays git
* Device: pc x86 

#### Issue
Wrong android checksum doesn't stop release process.
Information that it should be re-download is hidden deep in logs

#### Steps to reproduce
> ANDROID_ARCH=x86 ./kodev release android

CTRL+C when is downloading android-sdk , and then retry:

> ANDROID_ARCH=x86 ./kodev release android

##### `crash.log` (if applicable)
> [ -e android-sdk_r24.4.1-linux.tgz ] || wget https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
CHECKSUM=`md5sum android-sdk_r24.4.1-linux.tgz | awk '{ print $1 }'`; \
         [[ "$CHECKSUM" = "978ee9da3dda10fb786709b7c2e924c0" ]] || \
         { echo "android-sdk_r24.4.1-linux.tgz has the wrong checksum!"; exit 1; }
android-sdk_r24.4.1-linux.tgz has the wrong checksum!
Makefile:10: recipe for target 'android-sdk' failed
make: *** [android-sdk] Error 1
make: Leaving directory '/media/marcin/Dane/koreader/base/toolchain'
make androidupdate
make[1]: Entering directory `/media/marcin/Dane/koreader'
make -C base
make[2]: Entering directory `/media/marcin/Dane/koreader/base'
if ! test "" = "`cat kodebug-android 2>/dev/null`" ; then \
                echo "KODEBUG variable changed; calling luajit-clean"; \
                make luajit-clean; \
        fi; \
        echo "" >kodebug-android
install -d thirdparty/luasec/build/i686-linux-android
rm -f /media/marcin/Dane/koreader/base/thirdparty/luasec/build/i686-linux-android/luasec-prefix/src/luasec/../luasec-stamp/luasec-install

(.. and 1000 lines log more)

--- a/kodev
+++ b/kodev
@@ -178,6 +178,7 @@ ${SUPPORTED_TARGETS}"
             fi
             [ -e "${CURDIR}/base/toolchain/android-toolchain-${ANDROID_ARCH}/bin/" ] || {
                 { [ -e "${NDK}" ] || make -C "${CURDIR}/base/toolchain" android-ndk; }
+                assert_ret_zero $?
                 make android-toolchain
                 assert_ret_zero $?
             }

